docs/falco-update.md

```diff
@@ -0,0 +1,56 @@
+## Updating The Falco Fork
+
+### Useful Links:
+
+- Fork: https://github.com/stackrox/falcosecurity-libs
+- Upstream: https://github.com/falcosecurity/libs
+
+### Update Process
+
+The process to update the Falco fork is a branching strategy that essentially
+rebases our changes onto the latest Falco tag (not to the tip of master) in
+order to retain versions of released code and avoid maintenance hell.
+
+The process, at a high level is:
+
+- Update the `upstream-main` branch to the tip of master
+- Create a new module version branch from the previous version
+- Rebase the new module version branch to a newer tag on `upstream-main`
+    - This is likely to require conflict resolution and testing
+- Push the new module version branch to origin
+
+Or, described in Git commands:
+
+First you must identify the latest version branch in the repository:
+
+```
+$ git branch -a | grep 'module-version-'
+# e.g.
+module-version-2.9
+module-version-2.10
+```
+
+With a latest version in hand, you can proceed with the update:
+
+```sh
+$ git remote add falco git@github.com:falcosecurity/libs
+$ git fetch falco
+$ git checkout upstream-main
```

Comment:
Would it make sense to go modern and use `switch` here [[1]]?

[1]: https://git-scm.com/docs/git-switch